src/checkout/context/checkout.tsx

```diff
@@ -249,6 +252,13 @@ export const CheckoutProvider: FC<Props> = React.memo(({children}) => {
     return errs.length
   }
 
+  const isPaygCreditActive = useMemo(() => {
+    const query = new URLSearchParams(window.location.search)
+
+    const hasSignup = query.get('signup')
```

Comment:
do we wanna validate this against anything? Right now:

`?signup=foo`, `?signup=bar`, & `?signup=influxdataisthebest` would all be considered a signup

Reply:
Any value for signup should be okay. If you think we should be strict, then i can add the validation.

Reply:
It should also be noted that anyone can insert a query parameter and mock this situation out, so not sure how solid of a solution this is.